### PR TITLE
add delete button to shared space folder (#593)

### PR DESCRIPTION
**Pull request by imGok**
deletefolder

### CODE REVIEW

File: lib/presentation/widget/shared_space_document/shared_space_document_widget.dart [hunk 1]
```diff
@@ -629,16 +629,11 @@ class _SharedSpaceDocumentWidgetState extends State<SharedSpaceDocumentWidget> {
...
-                    _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder))
-                : sharedSpaceDocumentViewModel.openWorkGroupNodeContextMenu(
-                    context,
-                    node.element,
-                    _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
-                    footerAction: SharedSpaceOperationRole.deleteNodeSharedSpaceRoles.contains(_arguments?.sharedSpaceNode.sharedSpaceRole.name) ? _removeWorkGroupNodeAction([node.element]) : SizedBox.shrink())),
+            onPressed: () => sharedSpaceDocumentViewModel.openWorkGroupNodeContextMenu(
+              context,
+              node.element,
+              node.element.type == WorkGroupNodeType.FOLDER ? _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder) : _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
```
Comment (huynguyennovem) [inline lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: What's the difference? Why this change helps to fix the bug?

Comment (huynguyennovem) [reply lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: Should be break the line of code

Comment (imGok) [reply lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: the footer was not added when it was folders before, I added it and I refactored a little bit so it's more understandable

Comment (huynguyennovem) [reply lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: Oh, it's the footer, my eyes are so bad, LOL

Comment (huynguyennovem) [reply lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: But it still needs to break the line IMO and fix the build failed too.

Comment (hoangdat) [reply lib/presentation/widget/shared_space_document/shared_space_document_widget.dart:635]: Failed build, still by Certificate. Please ignore. fix the line
```
node.element.type == WorkGroupNodeType.FOLDER 
   ? _contextMenuFolderActionTiles(context, node.element as WorkGroupFolder)
   : _contextMenuDocumentActionTiles(context, node.element as WorkGroupDocument, indexWorkGroupDocument),
```

**Comment by imGok** [comment]
@hoangdat rdy to go

**Review by huynguyennovem — APPROVED** [review]
